packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -271,24 +345,41 @@ export default class SingleSelect extends React.Component<Props, State> {
     }
 
     render() {
-        const {alignment, dropdownStyle, light, style} = this.props;
-        const items = this.getMenuItems();
-        const opener = this.renderOpener(items.length);
+        const {
+            alignment,
+            children,
+            dropdownStyle,
+            isFilterable,
+            light,
+            style,
+        } = this.props;
+        const {searchText} = this.state;
+        const allChildren = React.Children.toArray(children).filter(Boolean);
+        const filteredItems = this.getMenuItems(allChildren);
+        const opener = this.renderOpener(allChildren.length);
 
         return (
             <DropdownCore
                 role="listbox"
                 alignment={alignment}
-                dropdownStyle={[selectDropdownStyle, dropdownStyle]}
+                dropdownStyle={[
+                    isFilterable && filterableDropdownStyle,
+                    selectDropdownStyle,
+                    dropdownStyle,
+                ]}
                 initialFocusedIndex={this.selectedIndex}
-                items={items}
+                items={[...this.getSearchField(), ...filteredItems]}
```

Comment:
style nit: It feels a little weird to have `getSearchField` return an array.  Maybe we could do:
```
const searchField = this.getSearchField();
const items = searchField ? [searchField, ...filteredItems] : filteredItems;
...
items={items}
```